### PR TITLE
feat: add default typedefs

### PR DESCRIPTION
**Pull request by LucasMull**

### CODE REVIEW

File: json-struct.c [hunk 1]
```diff
@@ -1663,9 +1663,7 @@ static void gen_struct(FILE *fp, struct jc_struct *s)
   }
   fputs(" */\n", fp);
 
-  if (t_alias)
-    fprintf(fp, "typedef ");
-  fprintf(fp, "struct %s {\n", t);
+  fprintf(fp, "typedef struct %s {\n", t);
```
Comment (mlite) [inline json-struct.c:1666]: do this mean every `struct` has an alias?

Comment (LucasMull) [reply json-struct.c:1666]: Yes, this change adds a default typedef if a alias isn't specified.

**Comment by mlite** [comment]
Why? It makes the spec language less expressive.

**Comment by LucasMull** [comment]
Ok thats a good point, I'll close this PR. Some users were complaining of having to write `struct` everytime so I went with the fastest route where we wouldn't have to edit any of the existing specs.

**Comment by mlite** [comment]
typedef is debatable.  typing `struct` isn't a problem when we use auto
code completion.  The most time consuming part of software development is
not typing.

**Pull request closed without merging**